cmake/FindTarantool.cmake

```diff
@@ -9,7 +9,8 @@ macro(extract_definition name output input)
 endmacro()
 
 find_path(TARANTOOL_INCLUDE_DIR tarantool/module.h
-  HINTS ENV TARANTOOL_DIR /usr/local/include
```

Comment:
Why have you removed /usr/local/include from consideration? What is the order supposed to be used if there is _no_ `$TARANTOOL_DIR` defined in the environment?

Reply:
I think cmake internal hard-coded find_path hints